Example/Backpack/NavigationData.swift

```diff
@@ -43,20 +43,20 @@ class NavigationData: NSObject {
             Item(name: "Bar charts", value: .story(loadStoryboard(name: "BarCharts", identifier: "BarChartsViewController")))
             Item(name: "Bottom sheet", value: .story(loadStoryboard(name: "BottomSheet", identifier: "BottomSheetViewController")))
             Group(name: "Buttons") {
-              ButtonStory.storyStructure
+              ButtonStory.allExamples
```

Comment:
I felt that 'examples' was better syntax here as otherwise it gets a bit confusing if we keep saying 'stories'. Like with web Storybook, each story can have one or more examples contained within.